classes/search/pattern/class-patternusageapi.php

```diff
@@ -77,12 +76,9 @@ private function fetch_items(): void {
 	 * Get all patterns names
 	 */
 	private function pattern_names(): array {
-		return array_filter(
```

Comment:
I wasn't sure how to solve this issue, but I figured this was an OK fix for now since we already have [an open PR](https://github.com/greenpeace/planet4-master-theme/pull/2303) to move this report to the theme